src/ucp/api/ucp_def.h

```diff
@@ -337,4 +337,6 @@ typedef enum ucp_wakeup_event_types {
     UCP_WAKEUP_TAG_RECV    = UCS_BIT(3)  /**< Tag receive completion */
 } ucp_wakeup_event_t;
 
+typedef struct ucp_request              ucp_request_t;
```

Comment:
There is a ucp_request_t in UCX master that is defined differently.

Reply:
It is defined in src/ucp/core/ucp_types.h which is not a public header. 
Thus the define here, since we are exposing the ucp_request_t in a public API. Not sure how to go about it differently.